example/main.tf

```diff
@@ -85,6 +85,38 @@ resource "google_compute_subnetwork" "vpc_subnetwork" {
   ]
 }
 
+resource "google_compute_router" "router" {
+  name    = "${var.router_name}"
+  network = "${google_compute_network.vpc_network.self_link}"
+
+  bgp {
+    asn = 64514
+  }
+}
+
+resource "google_compute_address" "address" {
+  count = 1
+  name  = "${var.address_name}-${count.index}"
+}
+
+resource "google_compute_router_nat" "nat" {
+  name                               = "${var.nat_name}"
+  router                             = "${google_compute_router.router.name}"
+  nat_ip_allocate_option             = "MANUAL_ONLY"
+  nat_ips                            = ["${google_compute_address.address.*.self_link}"]
+  source_subnetwork_ip_ranges_to_nat = "LIST_OF_SUBNETWORKS"
+
+  subnetwork {
+    name                    = "${google_compute_subnetwork.vpc_subnetwork.self_link}"
+    source_ip_ranges_to_nat = ["ALL_IP_RANGES"]
```

Comment:
Are there other options worth specifying here? If so, comment/making it an option would be great 😄

Reply:
This has also been updated and is commented in the new version.

I've not made it an option as the example project is just intended to show how the module can be used and I don't want to overcomplicate it with lots of options.